.github/workflows/main.yml

```diff
@@ -12,18 +12,22 @@ on:
 
 jobs:
   test:
-    name: Test python-${{ matrix.python-version }} sqla ${{ matrix.sqla-version }} on ${{ matrix.os }}
+    name: "Test python-${{ matrix.python-version }}
+     sqla: ${{ matrix.sqla-version }}
+     CrateDB: ${{ matrix.crate-version }}
+     on ${{ matrix.os }}"
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        crate-version: [nightly]
```

Comment:
can be extended with `testing` and `stable` https://github.com/crate/crate-python/pull/366/files#diff-90829e76e906f1c73140c7ded7e1b268R48-R54 will handle that.